scene/resources/2d/skeleton/skeleton_modification_2d_physicalbones.cpp

```diff
@@ -67,7 +67,7 @@ bool SkeletonModification2DPhysicalBones::_get(const StringName &p_path, Variant
 #ifdef TOOLS_ENABLED
 	if (Engine::get_singleton()->is_editor_hint()) {
 		if (path.begins_with("fetch_bones")) {
-			return true; // Do nothing!
+			return false; // Do nothing!
```

Comment:
this change seems wrong, will investigate

Reply:
OK this bool is basically a button in the inspector. Kinda hacky, but that means it should exist as a property but always return "false" I think.